company-profiles/xp-inc.md

```diff
@@ -0,0 +1,30 @@
+# XP Inc.
+
+## Company blurb
+
+XP Inc.'s purpose is to transform the financial market to improve people's lives. We started in 2001, as a small office for independent investment agents in Porto Alegre. We got here with the dedication of talented people who, through our culture of Big Dream, Open Mind and Entrepreneurial Spirit, had the autonomy and freedom they needed to transform the financial market.
+
+## Company size
+
+1001-5000
+
+## Remote status
+
+100% Remote
+
+## Region
```

Comment:
Hey @anamanzandev - can you confirm if these are the only countries where employees can be hire? 

See this line for clarification - https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md#region

Reply:
Hi @dougaitken ,

According to their LinkedIn and website, these are the regions they have employees, and they will have some of them working at Portugal soon.

Reply:
Thanks @anamanzandev, the reason I ask is that the Region should be where people can be employed, not where they currently have employees.